addons/clones/addon.json

```
@@ -47,6 +47,6 @@
     "version": "1.31.0",
     "newSettings": ["projectpage"]
   },
-  "tags": ["editor", "projectPlayer", "recommended"],
+  "tags": ["editor", "recommended"],
```

Clone counter works on the website, too.
```suggestion
  "tags": ["player", "recommended"],
```

---

It does now, but I think it's disabled by default outside the editor. I agree it should have the tag though.